Cargo.toml

```diff
@@ -19,11 +19,12 @@ license = "MIT"
 default = []
 avoid_timestamping = []
 jack = ["jack-sys", "libc"]
+winjack = ["jack"]
```

Comment:
I did it this way to avoid surprises/breaks for any downstream crates that have the `jack` feature enabled, but expect/handle a non-jack backend on Windows. With only `jack` enabled, Windows will still use the winmm backend. I'm open to just reusing `jack` if this isn't a concern (I think it would clean up some of the cfg guards)

Reply:
It would be even better if midir could pick between backends at runtime. Of course, that's a lot more work.

Reply:
Given that a semver breaking release is already being prepared, should we go ahead just use the `jack` feature for windows?